testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/security/authentication/BasicAuthenticationMechanismPicketboxRemovedTestCase.java

```diff
@@ -0,0 +1,188 @@
+import org.jboss.as.test.integration.web.security.authentication.deployment.SecuredEJBServlet;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.dmr.ModelNode;
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.net.URL;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.fail;
+
+/**
+ * Automated test for [ WFLY-10282 ]
+ * Test Enables elytron and removes security subsystem. After secured EJB is deployed and accessed using basic authorization
+ * test checks if no NullPointerException is thrown.
+ *
+ * @author Daniel Cihak
+ */
+@RunWith(Arquillian.class)
+public class BasicAuthenticationMechanismPicketboxRemovedTestCase {
+
+    private static Logger LOGGER = Logger.getLogger(BasicAuthenticationMechanismPicketboxRemovedTestCase.class);
+
+    private static final String EJB_SECURITY = "ejb-security";
+
+    private volatile ModelControllerClient modelControllerClient;
+    private volatile CommandContext commandCtx;
+    private volatile ByteArrayOutputStream consoleOut = new ByteArrayOutputStream();
+
+    @Deployment(name = EJB_SECURITY)
+    public static WebArchive appDeployment1() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, EJB_SECURITY + ".war");
+        war.addClasses(BasicAuthenticationMechanismPicketboxRemovedTestCase.class, SecuredEJB.class, SecuredEJBServlet.class);
+        war.setWebXML(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/web.xml"));
+        war.addAsWebInfResource(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/jboss-web.xml"), "jboss-web.xml");
+        war.addAsWebInfResource(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/beans.xml"), "beans.xml");
+        war.addAsResource(BasicAuthenticationMechanismPicketboxRemovedTestCase.class.getPackage(), "jboss-ejb-client.properties", "jboss-ejb-client.properties");
+
+        return war;
+    }
+
+    @Before
+    public void before() {
+        try {
+            initServerConfiguration(EJB_SECURITY);
+        } catch (Throwable t) {
+            LOGGER.error("Server configuration initialization failed.", t.getMessage(), t);
+            fail("Throwable:" + t.getMessage());
+        }
+    }
+
+    private void initServerConfiguration(String deployment) throws Exception {
+        modelControllerClient = ModelControllerClient.Factory.create(TestSuiteEnvironment.getServerAddress(), TestSuiteEnvironment.getServerPort());
```

Comment:
Why are you using the CLI and not the standard ServerSetupTask + ModelControllerClient?